packages/compass-smoke-tests/src/installers/linux-tar.ts

```diff
@@ -18,7 +18,14 @@ export function installLinuxTar({
   execute('tar', ['-xzvf', filepath, '-C', sandboxPath]);
 
   // Check that the executable will run without being quarantined or similar
-  execute('xvfb-run', [path.resolve(appPath, appName), '--version']);
+  // Passing --no-sandbox because RHEL and Rocky usually run as root and --disable-gpu to avoid warnings
```

Comment:
This comment is a bit confusing as I don't expect us to run this on RHEL anyway?